components/sensor/sgp40.rst

```diff
@@ -0,0 +1,57 @@
+SGP40 Volatile Organic Compound Sensor
+==============================================
+
+.. seo::
+    :description: Instructions for setting up SGP40 Volatile Organic Compound sensor
+    :image: sgp40.jpg
+
+The ``sgp40`` sensor platform  allows you to use your Sensiron SGP40 VOC sensor
+(`datasheet <https://www.sensirion.com/fileadmin/user_upload/customers/sensirion/Dokumente/9_Gas_Sensors/Sensirion_Gas_Sensors_SGP40_Datasheet.pdf>`__) with ESPHome.
+The :ref:`I²C Bus <i2c>` is required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/sgp40.jpg
+    :align: center
+    :width: 80.0%
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+      - platform: sgp40
+        name: "Workshop VOC"
+        update_interval: 5s
+        store_baseline: "true"
+        compensation:
+          humidity_source: dht1_temp
+          temperature_source: dht1_hum    
+
+
+Configuration variables:
+------------------------
+
+- **name** (**Required**, string): The name for the CO₂eq sensor.
+- **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor for use in lambdas.
+- **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
```

Comment:
Indicate default values on all optional scalar elements. should be `"Defaults to ``<value>``."` and  must obviously match with the value on the `py` file.

Reply:
Done